core-web/libs/edit-content/src/lib/fields/dot-edit-content-binary-field/components/dot-binary-field-editor/languages.ts

```diff
@@ -0,0 +1,178 @@
+const HTML_BASE_TOKENIZER_ROOT = [
```

Comment:
@zJaaal I created this for WYSIWYG a time ago. Maybe we need to merge it https://github.com/dotCMS/core/blob/1da9702704122e33b1d75a22d4767401a3fc9d48/core-web/libs/edit-content/src/lib/fields/dot-edit-content-wysiwyg-field/components/dot-wysiwyg-monaco/custom-languages/velocity-monaco-language.ts